RecoMET/METAlgorithms/interface/HaloClusterCandidateEB.h

```diff
@@ -0,0 +1,97 @@
+#ifndef RECOMET_METALGORITHMS_HALOCLUSTERCANDIDATEEB_H
+#define RECOMET_METALGORITHMS_HALOCLUSTERCANDIDATEEB_H
+
+
+
+#include "DataFormats/DetId/interface/DetId.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/EcalDetId/interface/EBDetId.h"
+#include "DataFormats/EcalDetId/interface/EEDetId.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitFwd.h"
+#include "DataFormats/GeometrySurface/interface/Cylinder.h"
+#include "DataFormats/GeometrySurface/interface/Plane.h"
+#include "DataFormats/GeometrySurface/interface/Cone.h"
+#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
+#include "DataFormats/GeometryVector/interface/GlobalVector.h"
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+#include "DataFormats/HcalRecHit/interface/HcalRecHitCollections.h"
+#include "DataFormats/EcalRecHit/interface/EcalRecHitCollections.h"
+#include "DataFormats/Math/interface/deltaPhi.h"
+#include "Geometry/CaloGeometry/interface/CaloGeometry.h"
+#include "Geometry/Records/interface/CaloGeometryRecord.h"
+#include "FWCore/Framework/interface/Event.h"
+
+
+
+class HaloClusterCandidateEB {
+
+ public:
+  HaloClusterCandidateEB(){
+    et =0;
+    seed_et =0;
+    seed_eta =0;
+    seed_phi =0;
+    seed_Z =0;
+    seed_R =0;
+    seed_time =0;
```

Comment:
the amount of copy-paste replicated code would reduce substantially if seed {et,eta, phi, z, r, time} are wrapped in a separate class.